docs/serving/expert_parallel_deployment.md

```diff
@@ -123,12 +123,22 @@ When enabled, vLLM collects load statistics with every forward pass and periodic
 
 ### EPLB Parameters
 
+The new `--eplb-config` argument should be used, which accepts a JSON string. The available keys and their descriptions are:
```

Comment:
Small nit since we no longer mention the old way

```suggestion
Configure EPLB with the `--eplb-config` argument, which accepts a JSON string. The available keys and their descriptions are:
```